data-prepper-plugins/opensearch/README.md

```diff
@@ -144,6 +144,20 @@ If a single record turns out to be larger than the set bulk size, it will be sen
 
 - `ism_policy_file` (optional): A String of absolute file path for an ISM (Index State Management) policy JSON file. This policy file is effective only when there is no built-in policy file for the index type. For example, `custom` index type is currently the only one without a built-in policy file, thus it would use the policy file here if it's provided through this parameter. OpenSearch documentation has more about [ISM policies.](https://opensearch.org/docs/latest/im-plugin/ism/policies/)
 
+### Management Disabled Index Type
```

Comment:
QUES: From the test case, is index creation a prerequisite for using this type? If yes, we should specify here.

Reply:
Yes, it is a prerequisite. I can make it clearer.